strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/AuthorizationConfigController.java

```diff
@@ -147,88 +139,187 @@ public ResponseEntity getAuthorizationConfig()
     @ApiOperation(value = "Deletes a role by name.",
                   position = 3)
     @ApiResponses(value = { @ApiResponse(code = 200,
-                                         message = "The role was deleted."),
+                                         message = SUCCESSFUL_DELETE_ROLE),
                             @ApiResponse(code = 400,
-                                         message = "Bad request.")
+                                         message = FAILED_DELETE_ROLE)
     })
-    @RequestMapping(value = "role/{name}",
-                    method = RequestMethod.DELETE)
+    @DeleteMapping(value = "/role/{name}",
+                   consumes = MediaType.APPLICATION_JSON_VALUE,
```

Comment:
This mapping doesn't actually consume anything here as it gets the name from the url - you can remove the `consumes`.